app/src/main/java/org/mozilla/fenix/home/recentsyncedtabs/RecentSyncedTabFeature.kt

```diff
@@ -106,12 +117,19 @@ class RecentSyncedTabFeature(
                     url = tab.url,
                     iconUrl = tab.iconUrl
                 )
-            } ?: return
-        recordMetrics(syncedTab, lastSyncedTab, syncStartId)
-        appStore.dispatch(
-            AppAction.RecentSyncedTabStateChange(RecentSyncedTabState.Success(syncedTab))
-        )
-        lastSyncedTab = syncedTab
+            }
+
+        if (syncedTab == null) {
+            appStore.dispatch(
+                AppAction.RecentSyncedTabStateChange(RecentSyncedTabState.None)
+            )
+        } else {
+            recordMetrics(syncedTab, lastSyncedTab, syncStartId)
+            appStore.dispatch(
+                AppAction.RecentSyncedTabStateChange(RecentSyncedTabState.Success(syncedTab))
+            )
+            lastSyncedTab = syncedTab
+        }
```

Comment:
Yeah, it seems like this prevents the loading placeholder from appearing at all. It sounded like product was okay with removing the placeholder, but I would make sure to inform design as well. I personally prefer it over having the card pop-in, but I will leave it up to you

Reply:
I see.  I'll update the solution here so we don't have placeholder for now.  This could be risky if we see the placeholder card pop in and out.  It will make my change smaller and at the same time less risky.